tests/unit/test_worker.py

```diff
@@ -20,5 +20,5 @@ def test_worker_decoder_handles_pydantic_models(authors: list[Author]) -> None:
     encoded = worker.encoder.encode(pydantic_model)
     assert (
         encoded
-        == b'{"id":"97108ac1-ffcb-411d-8b1e-d9183399f63b","created":"0001-01-01T00:00:00","updated":"0001-01-01T00:00:00","name":"Agatha Christie","dob":"1890-09-15"}'
+        == b'{"created":"0001-01-01T00:00:00","updated":"0001-01-01T00:00:00","id":"97108ac1-ffcb-411d-8b1e-d9183399f63b","name":"Agatha Christie","dob":"1890-09-15"}'
```

Comment:
I tried settings the id column position to 1 (`id._column_position = 1`), but the mixin still listed the audit columns first.

Reply:
Maybe something like this: https://stackoverflow.com/a/4013184/6560549